testsupport/wait/host.go

```diff
@@ -1734,7 +1733,7 @@ func (a *HostAwaitility) CreateAPIProxyConfig(t *testing.T, usertoken, proxyURL
 func (a *HostAwaitility) CreateAPIProxyClient(t *testing.T, userToken, proxyURL string) (client.Client, error) {
 	proxyKubeConfig := a.CreateAPIProxyConfig(t, userToken, proxyURL)
 
-	s := scheme.Scheme
+	s := runtime.NewScheme()
```

Comment:
let's make sure that there are no other cases where we could face the same issue

Reply:
just to be sure, I would change this one too: https://github.com/codeready-toolchain/toolchain-e2e/blob/78c7ae131fc87f5f35cceb124c9a6afad81c7390/testsupport/init.go#L262

Reply:
there are 2-3 other places, which i intend to change but, i think it might  require a bit of discussion, hence if this fix is working, we can merge this and fix our broken ci.. and after that i can open a Pr with the others.

Reply:
updated it